ale_linters/java/eclipselsp.vim

```diff
@@ -76,6 +77,16 @@ function! ale_linters#java#eclipselsp#CommandWithVersion(buffer, version_lines,
     return ale_linters#java#eclipselsp#Command(a:buffer, l:version)
 endfunction
 
+function! ale_linters#java#eclipselsp#WorkspacePath(buffer) abort
+    let l:wspath = ale#Var(a:buffer, 'java_eclipselsp_workspace_path')
+
+    if !empty(l:wspath)
+        return l:wspath
+    endif
+
+    return ale#path#Simplify(ale#java#FindProjectRoot(a:buffer) . '/..')
```

Comment:
You can use `return ale#path#Dirname(ale#java#FindProjectRoot(a:buffer))`. It gets the dirname for a path and handles the path being empty.

Reply:
Changed to use Dirname.